app/views/content/_table_data_description.html.erb

```diff
@@ -2,6 +2,8 @@
     <span class="table-header__param"><%= pagination.first_record %></span> to
     <span class="table-header__param"><%= pagination.last_record %></span> of
     <span class="table-header__param"><%= pagination.total_results %></span> results
-    from <span class="table-header__param">UK Visas and Immigration</span>
-    in <span class="table-header__param">guidance</span>.
+    from <span class="table-header__param"><%= filter.organisation_name %></span>
+    <% if filter.document_type? %>
+        in <span class="table-header__param"><%= filter.document_type %></span>.
+    <% end %>
```

Comment:
Had a discussion with @pollygreen, the current wording and order doesn't make sense. 
e.g.
Showing **1** to **100** of **12245**  results from **UK Visas and Immigration** in **Guidance** 

Could we re-order it reflect this?
Showing **1** to **100** of **12245**  results for **Guidance** from **UK Visas and Immigration**